scout/server/config.py

```diff
@@ -97,6 +97,8 @@
 SHOW_CAUSATIVES = True
 SHOW_OBSERVED_VARIANT_ARCHIVE = True
 HIDE_ALAMUT_LINK = True
+# Display case rerun monitoring toggle
+RERUN_MONITOR = True
```

Comment:
```suggestion
# RERUN_MONITOR = True
```

Reply:
Just an idea: since other centers are not likely going to have a use for the functionality just now, we could switch if off in the demo settings?

Reply:
Hm, this is the demo config. I would suggest keeping it enabled as possible, when other components are not really invoked. I mean, a test user could run the cli export command for monitored cases. But it doesn't matter very much either way.

Reply:
Ah, but that reminds me - what happens if the key does not exist! Will fix!

Reply:
Ok, see, my comment has been useful anyway! :)

Reply:
> Ok, see, my comment has been useful anyway! :)

Essentially always! 😸